core/storage/trie/polkadot_trie_db/polkadot_trie_batch.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_STORAGE_TRIE_POLKADOT_TRIE_DB_POLKADOT_TRIE_BATCH_HPP
+#define KAGOME_CORE_STORAGE_TRIE_POLKADOT_TRIE_DB_POLKADOT_TRIE_BATCH_HPP
+
+#include "common/buffer.hpp"
+#include "storage/face/write_batch.hpp"
+#include "storage/trie/polkadot_trie_db/polkadot_trie_db.hpp"
+
+namespace kagome::storage::trie {
+  class PolkadotTrieBatch
+      : public face::WriteBatch<common::Buffer, common::Buffer> {
+    enum class Action { PUT, REMOVE };
+    struct Command {
+      Action action;
+      common::Buffer key;
```

Comment:
probably you need to have {} on both members?
```suggestion
      common::Buffer key{};
```

Reply:
Key is a mandatory member, whereas value is not.

Reply:
only matter of clang-tidy, if it's ok, I don't mind

Reply:
Anyway, pls make them look the same.

Reply:
It's not a matter of clang-tidy, attribute of struct with brackets is going to be default initialized, and you don't need to explicitly do it when aggregate initializing it, so the brackets should be there

Reply:
I don't think it matters much in this case. Added them to be more consistent.